### ISSUE TITLE
Updating Make

### ISSUE THREAD

**Issue by georgydunaev** [coq-contribs/zfc#4]
According to 

https://stackoverflow.com/questions/53114522/development-of-the-coq-library-add-loadpath-solution-is-not-good-enough

it is good practice to add "From ZFC " to every "Require Import", rename filename "Make" to "_CoqProject" everywhere, and deleting the first line "-R . ZFC".

By the way, usual name is "zfc", not "ZFC".
I think we may 
a)change the name of github repository to "ZFC" (which is more reasonable variant, but something may go wrong, for example for my fork. Or may not.)
b)or change everywhere name to "zfc". "From zfc Require ...". (not very good solution, (a) is better).

**Comment by herbelin** [comment]
Yes, I prefer (a) too. And adding `From ZFC` is a good practice.

**Comment by Zimmi48** [comment]
Hi @georgydunaev, I think you have misunderstood @Lysxia's Stack Overflow answer: the first line `-R . ZFC` is really important, even if you rename `Make` into `_CoqProject`. Logical names (i.e. namespaces like `ZFC`) do not have to be the same as directory name (here `zfc`) although making them consistent is not a problem either. But even if you do, people will still be able to clone into a directory that would be differently named.
